test/Operators/finitedifference/opt.jl

```diff
@@ -237,7 +237,7 @@ end
             @test_opt opt_LeftBiasedF2C(faces)
             @test_opt opt_RightBiasedF2C(faces)
 
-            @test_opt opt_AdvectionF2F(face_velocities, faces)
+            # @test_opt opt_AdvectionF2F(face_velocities, faces)
```

Comment:
Can/should we uncomment?

Reply:
So it turns out that `AdvectionF2F` as it is currently defined doesn't work (i.e. it doesn't have well-defined boundary conditions)

Reply:
(and consequently, it now breaks a bunch of tests)

Reply:
Ok, I don't see it used in ClimaAtmos, fine with commenting